src/components/forms/AppSearchInput.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <form
+    class="relative"
+    @submit.prevent="emit('update:modelValue', searchText)"
+  >
+    <AppInput v-model="searchText" class="pr-8" :placeholder="placeholder" />
```

Comment:
Add `:inputType="text"` as it's not the default on this component?

Reply:
Ah I didn't realise, thanks. It feels like the correct behaviour would be that it defaults to `type="text"` as that would reflect the real HTML `input` element. What do you think?

Reply:
I feel the same way!